### PR TITLE
[Feature] Applicant labels

### LINKED ISSUE #574: [Feature] Applicant labels
- [x] Allow a "addition" selection of labels to an applicant
- [x] Allow color selection of labels
- [x] Labels should be remembered for each company
- [x] Add label management to company settings
  - [x] Addition
  - [x] Deletion
  - [x] Updating
- [x] Display labels on
  - [x] Applicant card (set a hard limit, display the rest in a hover menu)
  - [x] Applicant profile

### PR DESCRIPTION
**Pull request by petersonbrock**
closes #574

### CODE REVIEW

File: app/config/forms/lists.js [hunk 1]
```diff
@@ -0,0 +1,21 @@
+export const lists = {
+  'dqReasons': {
```
Comment (James1x0) [inline app/config/forms/lists.js]:
```suggestion
  dqReasons: {
```

File: app/config/forms/lists.js [hunk 2]
```diff
@@ -0,0 +1,21 @@
...
+    elements: [{
+      label: 'List Item',
+      type:  'text',
+      path:  'currentItem'
+    }],
+    listType: 'string'
+  },
+  'labels': {
```
Comment (James1x0) [inline app/config/forms/lists.js]:
```suggestion
  labels: {
```

File: app/controllers/account/job-opening/campaign/applicant-tracking.js [hunk 3]
```diff
@@ -4,6 +4,7 @@ import { A } from '@ember/array';
 import { inject as service } from '@ember/service';
 import RSVP from 'rsvp';
 import moment from 'moment';
+import $ from 'jquery';
```
Comment (James1x0) [inline app/controllers/account/job-opening/campaign/applicant-tracking.js]: Is this used?

File: app/controllers/application.js [hunk 4]
```diff
@@ -62,7 +62,7 @@ export default Controller.extend({
         backgroundRGB = backdrop.match(filterNumbers),
         [ r, g, b ] = backgroundRGB.map(Number);
 
-    return (r * 299 + g * 587 + b * 114) / 1000 > 150 ? true : false;
+    return (r * 299 + g * 587 + b * 114) / 1000 > 123 ? true : false;
```
Comment (James1x0) [inline app/controllers/application.js]: Just curious what this change is for

File: app/templates/components/card-pipeline/stage-card.hbs [hunk 5]
```diff
@@ -100,3 +118,5 @@
     </a>
   </div>
 {{/if}}
+
+{{!-- ADD ADD LABELS ACTION AND DISPLAY LABELS ON CARD --}}
```
Comment (James1x0) [inline app/templates/components/card-pipeline/stage-card.hbs]: Is this still a wip?

File: app/controllers/application.js (hunk 4 above)
Comment (petersonbrock) [reply app/controllers/application.js]: we decided at starbucks one morning that this was a better number for this formula

File: app/templates/components/card-pipeline/stage-card.hbs (hunk 5 above)
Comment (petersonbrock) [reply app/templates/components/card-pipeline/stage-card.hbs]: no, just forgot to take my reminder comment out

File: app/controllers/account/job-opening/campaign/applicant-tracking.js (hunk 3 above)
Comment (petersonbrock) [reply app/controllers/account/job-opening/campaign/applicant-tracking.js]: nope, my b

**Pull request merged**